### DISCUSSION TITLE
Question about this picture of wiring

### DISCUSSION THREAD

**Discussion by asotelles** [headtracker/HeadTracker#185] [Q&A]
Hello,
https://headtracker.gitbook.io/head-tracker/getting-started/wiring/option-3-bridged/radio-master-tx16s-mk2
On this page of wiki, blue wire goes to the tx of the radio instead of rx, is that right?

Other question, does radiomaster pocket has similar internal conector?

**Comment by dlktdr** [comment]
Yes, on the Mark II it's mislabeled on that connector internally. 

Not sure how about pocket. Hopefully somebody else can chime in

**Reply by asotelles** [reply]
thanks for the response
